Add support for shortener services (#191)
mautic/user-documentation: docs/configuration/shortener.rst

URL shortener service

> #################

Reviewer: ```suggestion
```

.. vale off

> Shortener service

Reviewer: ```suggestion
URL shortener service
```

I think it would be better to make this clearer that it's used for URLs.

> URL shortener service

Reviewer: ```suggestion
URL shortener service
```
You don't need to turn off Vale here, it turns it off for the entire page until you turn it back on again. That's cheating 😈

> #########

Reviewer: ```suggestion
#################
```

The underline must be at the same length as the text that it is beneath.

> 

Reviewer: ```suggestion
#####################
```

Heading underlines must be the same length as the text above them.

.. attention::

    Mautic 5 introduces a new shortening feature, replacing the previous legacy shortening system found in the settings. If you utilized the shortening service in Mautic 4, upon upgrading to Mautic 5, please follow the steps below to reconfigure it.

> The new shortener service allows developers to create plugins for any shortener service, with any type of authorization. By default, Mautic does not provide a shortener service plugin; you must install it either from the marketplace or manually.

Reviewer: ```suggestion
The new URL shortener service allows developers to create Plugins for any shortener service, with any type of authorization. By default, Mautic does not provide a URL shortener service Plugin; you must install it either from the Marketplace or manually.
```

Example of setup Bitly plugin

> =================

Reviewer: ```suggestion
=============================
```
The underline must be as long as the text above

> 1. Install the Bitly bundle from the marketplace or using Composer:

Reviewer: ```suggestion
1. Install the Bitly bundle from the Marketplace or using Composer:
```

.. code-block:: bash

    composer require webmecanik/mautic-bitly-bundle

> 2. Obtain an access key from https://app.bitly.com/settings/api and set up/enable the Bitly plugin.

Reviewer: ```suggestion
2. Obtain an access key from :xref:`Bitly API settings` and set up/enable the Bitly plugin.
```
If you need to use a link, please cd into the docs directory and use the command `make link` - follow the steps and commit the link file that is created. This hooks in with the automatic link checking linter and avoids duplication! I will commit the file to your PR to save you having to do it this time.
